tools/autograd/gen_python_functions.py

```diff
@@ -149,11 +149,20 @@ def should_bind(declaration):
 
 
 def gen_py_torch_functions(out, declarations):
+    def is_namespace_or_type_function(declaration):
```

Comment:
this name doesn't really match what it's doing (a decl only on the Type and Tensor I would expect to return True from the name); maybe instead of describing what the function does, describe what you are trying to achieve?

Reply:
were you planning to address this?